src/Table/Synapse/TableIndexDefinition.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Table\Synapse;
+
+/**
+ * @todo this class is missing validation for CLUSTERED INDEX now, but this is not yet used
+ */
+final class TableIndexDefinition
+{
+    public const TABLE_INDEX_TYPE_CCI = 'CLUSTERED COLUMNSTORE INDEX';
```

Comment:
navrhl bych přejmenovat ty suffixy CCI / CI. úplně to zavádí typo a průšvihem

Reply:
Jo roztáhnul sem ten název, good point